### PR TITLE
feat(directory): add address extra

### PR DESCRIPTION
**Pull request by JeremyDec**
### Description of the Change
Extra information of address added

### CODE REVIEW

File: client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <translation id="telephony_service_contact_urbandistrict">Arrondissement (pour Paris, Lyon et Marseille)</translation>
     <translation id="telephony_service_contact_waynumber">Numéro de voie</translation>
     <translation id="telephony_service_contact_wayname">Nom de la voie</translation>
+    <translation id="telephony_service_contact_addressextra">Complément d'adresse'</translation>
```
Comment (frenautvh) [inline client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml]: trailling quote

Comment (FredericEspiau) [inline client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml]: Remove the closing `'`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 2]
```diff
@@ -488,6 +488,23 @@
                         </span>
                     </div>
 
+                    <!-- ADDRESS EXTRA-->
+                    <div class="form-group">
+                      <label for="contactAddressExtra"
```
Comment (Jisay) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: can you please use 4 spaces for indent ?

File: .gitignore [hunk 3]
```diff
@@ -30,3 +30,4 @@ server/certificate/server.crt
 server/certificate/server.csr
 server/certificate/server.key
 tools/
+package-lock.json
```
Comment (Jisay) [inline .gitignore]: please do not ignore this file. It's needed to go faster on CDS ;-)

File: client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml (hunk 1 above)
Comment (JeremyDec) [reply client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml]: done

File: client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml [hunk 4]
```diff
@@ -26,6 +26,7 @@
     <translation id="telephony_service_contact_urbandistrict">Arrondissement (pour Paris, Lyon et Marseille)</translation>
     <translation id="telephony_service_contact_waynumber">Numéro de voie</translation>
     <translation id="telephony_service_contact_wayname">Nom de la voie</translation>
+    <translation id="telephony_service_contact_addressextra">Complément d'adresse</translation>
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml]: `'` => `’`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html (hunk 2 above)
Comment (antleblanc) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `data-translate` could be directly on the `<label>` element.

File: client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml (hunk 4 above)
Comment (JeremyDec) [reply client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml]: same keychar as all others `'` in the file, for example : "Publication dans l'annuaire inversé", should i replace all others ? Or do i leave it as it ?

**Pull request merged**